src/connection.rs

```diff
@@ -216,7 +207,7 @@ pub fn get_connections(dbus_manager: &Rc<DBusNetworkManager>) -> Result<Vec<Conn
 
 
 pub fn get_active_connections(dbus_manager: &Rc<DBusNetworkManager>)
-                              -> Result<Vec<Connection>, String> {
+    -> Result<Vec<Connection>, String> {
```

Comment:
This one is caused by `fn_args_layout = "Block"`, which positively affects function declarations with more arguments, but in this particular case not so.